components/micro-gateway-core/src/main/ballerina/gateway/endpoints/etcd_endpoint.bal

```diff
@@ -0,0 +1,29 @@
+// Copyright (c)  WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/io;
+import ballerina/http;
+import ballerina/config;
+
+endpoint http:Client etcdEndpoint {
+    url: retrieveConfig("etcdurl", "http://127.0.0.1:2379"),
```

Comment:
I think the URL mentioned here ( http://127.0.0.01:2379") would be subjected to changes. If that is the case, hard-coding such a value would not be the ideal solution here.